src/com/zenuml/sequence/plugins/jetbrains/license/CheckLicense.java

```diff
@@ -11,84 +10,80 @@
 import java.security.cert.*;
 import java.util.*;
 
-
-/**
- * @author Eugene Zhuravlev
- * Date: 26-Jul-18
- */
 public class CheckLicense {
   /**
-   * PRODUCT_CODE must be the same specified in plugin.xml inside the <productCode> tag
+   * PRODUCT_CODE must be the same specified in plugin.xml inside the <product-descriptor> tag
    */
-  private static final String PRODUCT_CODE = "PZENUML";
+  private static final String PRODUCT_CODE = "SMPL";
   private static final String KEY_PREFIX = "key:";
   private static final String STAMP_PREFIX = "stamp:";
+  private static final String EVAL_PREFIX = "eval:";
 
   /**
    * Public root certificates needed to verify JetBrains-signed licenses
    */
   private static final String[] ROOT_CERTIFICATES = new String[]{
-    "-----BEGIN CERTIFICATE-----\n" +
```

Comment:
Is this only format change? If that is the case, it should be reverted or commit in a different change set.

Reply:
I didn't notice ,  yes it's only space change (4 to 8).

the code was copied from jetbrains directly.  so was the previous version.
And we do not use this CheckLicense class at all.

it should be fine to keep the version that jetbrains provides